src/ontology/config/README.md

```diff
@@ -0,0 +1,15 @@
+## Codebook: `src/ontology/config/`
```

Comment:
Added some documentation for these files.

Reply:
good idea